pandas/core/sparse/frame.py

```diff
@@ -846,6 +846,13 @@ def applymap(self, func):
         return self.apply(lambda x: lmap(func, x))
 
 
+def _nan_to_np_nan(value):
+    """Normalize nan values to singleton np.NaN object so that when NaNs are
```

Comment:
have a look thru how DataFrame handles this in the init_dict routines

don't want to be reinventing the wheel here